base/abstractset.jl

```diff
@@ -59,7 +59,7 @@ julia> a
 Set([7, 4, 3, 5, 1])
 ```
 """
-union!(s::AbstractSet, sets...) = foldl(union!, s, sets)
+union!(s::AbstractSet, sets...) = foldl(union!, sets; init = s)
```

Comment:
Generally we don't use spaces around `=` for keyword arguments.

Reply:
thanks